pkgs/os-specific/linux/ebpf-usb/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, bcc
+, makeWrapper
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ebpf-usb";
+  version = "unstable-2022-04-03";
+  format = "other";
```

I need to remember, but I think it was "deprecated":

 
```suggestion
  pyproject = false; 
```